src/TimeModeling/judiVector.jl

```diff
@@ -288,37 +288,8 @@ adjoint(a::judiVector{vDT}) where vDT =
 
 ##########################################################
 
-# Overload base function for SegyIO objects
-
+# Overload needed base function for SegyIO objects
 vec(x::SegyIO.SeisCon) = vec(x[1].data)
-norm(x::SegyIO.IBMFloat32; kwargs...) = norm(convert(Float32,x); kwargs...)
```

Comment:
Just curious, why aren't these needed anymore?

Reply:
They were never needed, not used anywhere. And if it where these belongs in SegyIO not JUDI